src/mintpy/objects/gps.py

```diff
@@ -508,10 +509,30 @@ def get_los_geometry(self, geom_obj, print_msg=False):
             az_angle  = ut.heading2azimuth_angle(float(geom_obj['HEADING']))
 
         else:
-            raise ValueError(f'input geom_obj is neight str nor dict: {geom_obj}')
+            raise ValueError(f'input geom_obj is neither str nor dict: {geom_obj}')
 
-        return inc_angle, az_angle
+        return inc_angle or np.nan, az_angle or np.nan
 
+    def get_image_values(self, filename, datasetName=None, pad: int = 0, print_msg=False):
```

Comment:
This function is not used currently anywhere, so it's fine with me. Just to be clear: from our experience, given a GNSS site, we either interpolate InSAR or average multiple InSAR pixels around the GNSS site. Personally, I have not gotten a good result yet by comparing GNSS and its nearest InSAR pixel yet.